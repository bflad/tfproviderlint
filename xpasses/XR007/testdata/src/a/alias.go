package a

import (
	e "os/exec"
)

var failingAlias = e.Command // want "avoid os/exec.Command"

func fAlias() {
	e.Command("true") // want "avoid os/exec.Command"

	failingAlias("true")
}
