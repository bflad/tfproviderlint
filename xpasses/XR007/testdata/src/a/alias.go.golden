package a

import (
	e "os/exec"
)

// want "avoid os/exec.Command"

func fAlias() {
	// want "avoid os/exec.Command"

	failingAlias("true")
}
